tests/support/pretest.js

```diff
@@ -88,6 +88,9 @@ function getFiles(tree) {
 
 function getTests(tree) {
 	return Promise.all(_.map(tree, function(object) {
+		if (object.type === 'tree') {
```

Comment:
The problem that we were encountering was the introduction of the `resources` directory inside the `pointerevents` directory. Since `fs.outputFileAsync()` automatically creates directories as necessary, we can just ignore all directories inside `pointerevents`.